cypress/test/search-column-selector.spec.ts

```diff
@@ -0,0 +1,53 @@
+import {SearchColumnSelectorSteps as step} from '../steps/search-column-selector-steps';
+
+describe('Onto Search Column Selector Component', () => {
+  it('Should find the component', () => {
+    step.visit();
+    step.getOntoColumnSelectorComponent().should('be.visible');
+  });
+
+  it('Should test column selector functionality', () => {
+    // When I visit the page with column selector
+    step.visitWithDefaultOptions();
+    // I should see the selector
+    step.getColumnSelectDropdown().should('be.visible');
+    // And the selected columns should be the default ('Username')
+    step.getSelectedColumns().should('contain', 'Username');
+    // When I press the select
+    step.getColumnSelectDropdown().click();
+    // I should see column groups
+    step.getColumnGroups().should('be.visible');
+    // And they should contain 'Default Columns' and 'Other Columns'
+    step.getColumnGroups().should('contain', 'Default Columns').and('contain', 'Other Columns');
+    // When I look at 'Default Columns Group' I should see columns 'Username' and 'Name'
+    step.getColumnGroups().filter('[appCypressData=column-group-0]')
+        .find('mat-option').should('contain', 'Username').and('contain', 'Name');
```

Comment:
You can create a method in the steps for obtaining concrete option by index maybe